sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1298,6 +1298,7 @@
       "allowCalculateOverride": false,
       "validate": {
         "required": true,
+        "onlyAvailableItems": false,

```

Comment:
Please change to true.

Reply:
Not part of my change, but will do it. This occurs as these forms are created before the formio upgrade